lib/survey_execution/initializer.rb

```diff
@@ -0,0 +1,13 @@
+module SurveyExecution
+  class Initializer
+    def initialize(survey, respondents)
+      @client = Sms::Client.instance
```

Comment:
Does instance return the class instance? Should this also grab the client object (i.e. `Sms::Client.instance.@client`)?